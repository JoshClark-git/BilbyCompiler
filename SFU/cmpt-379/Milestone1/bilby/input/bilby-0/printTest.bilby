main {
    imm nickels := 11;
    imm pennies := 5 + 41;
    imm dimes := 4;
    
    imm value := dimes * 10 +
    			 nickels * 5 +
    			 pennies;
    			 
    print value $s$s value $s$n$n value $n nickels $ pennies $n;
}