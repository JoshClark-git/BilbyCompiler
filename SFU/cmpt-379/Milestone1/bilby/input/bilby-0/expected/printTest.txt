141  141 

141
1146
